WireAnalytics/Sources/WireAnalytics/AnalyticEvent.swift

```diff
@@ -18,8 +18,9 @@
 
 /// All possible analytic events that can be tracked
 
-public enum AnalyticEvent: String {
+public protocol AnalyticEvent {
```

Comment:
For consistency, we probably should call this `AnalyticsEvent`.

Reply:
Done: https://github.com/wireapp/wire-ios/commit/c7dbc9a96d31bae301ac87e1f31b7d8cb4623c0f